folding-schemes/src/commitment/pedersen.rs

```diff
@@ -77,6 +92,7 @@ impl<C: CurveGroup, const H: bool> CommitmentScheme<C, H> for Pedersen<C, H> {
         Ok(params.h.mul(r) + C::msm_unchecked(&params.generators[..v.len()], v))
     }
 
+    // TODO (autoparallel): I'm guessing `_rng` is marked with prefix `_` because it goes unused always and it is planned for the future? Otherwise we should remove that prefix.
```

Comment:
In this case (Pedersen), it is not used, but in an other implementation of the `CommitmentScheme` trait (ie. IPA) it is used.